### DISCUSSION TITLE
OIDC/OAuth2 Login

### DISCUSSION THREAD

**Discussion by Hutch79** [seerr-team/seerr#995] [Ideas]
Hey there 👋 

I'm starting this discussion to gather feedback about the benefits and disadvantages of having OIDC/OAuth2 built in to Jellyseer.

Currently, the only external Login provider is Jellyfin itself.
And please don't get me wrong, that's already a pretty nice solution. But for Admins which like to overcomplicate stuff or actually need a centralized Login infrastructure like Keycloak or Authentik, such a feature could be pretty awesome.
Therefore, this discussion is mainly about the benefits for more advanced users which do have such a setup.

The biggest benefit from a User perspective would be the ability to log into Jellyseer before login in to Jellyfin.
Another benefit would be SSO.

From an Administration standpoint, it could mean that Jellyseer does not need an Admin account to do its job. 
Although, I'm not sure if it would be possible to check if a movie/series is available with a Jellyfin API Key. But even if an API key is not enough, a normal User account with permission to see all Library's would be sufficient to do so.

I totally get that such a feature will not be a high priority feature in development, since it only affects a small group of peoples who would actually use such a feature. But I wanted to get feedback on this idea anyway.
So feel free to comment about things I missed and also if you would enjoy such a feature or if you think that this is totally unnecessary.

Love to hear your feedback!

**Comment by gauthier-th** [comment]
There is a first attempt here: #184
SSO is also in [our roadmap](https://github.com/Fallenbagel/jellyseerr/issues/926)

**Reply by Hutch79** [reply]
Hey @gauthier-th
Thanks for the answer!

Wow, that PR is old ^^
But I'm glad to see that this is already on the roadmap.
